sheets/parted

```diff
@@ -1,4 +1,4 @@
-# create a new MBR/msdos partition table for BIOS systems
+# create a new MBR/MSDOS partition table for BIOS systems
```

Comment:
I think it's technically MS-DOS; that's at least how I saw it displayed, back in the day.